test/core/test_dlfcn_jspi_main.c

```diff
@@ -0,0 +1,29 @@
+// Copyright 2025 The Emscripten Authors.  All rights reserved.
+// Emscripten is available under two separate licenses, the MIT license and the
```

Comment:
Could you call this file just `test_dlfcn_jspi.c` , I think its useful to have the filename match the test name where possible.